build/tf-wrapper.sh

```diff
@@ -138,7 +138,11 @@ tf_validate() {
       cd "$path" || exit
       terraform show -json "${tmp_plan}/${tf_component}-${tf_env}.tfplan" > "${tf_env}.json" || exit 32
       if [[ "$policy_type" == "CLOUDSOURCE" ]]; then
-        gcloud source repos clone gcp-policies "${policy_file_path}" --project="${project_id}" || exit 34
+        if [ ! -f /workspace/.policies-cloned ]; then
+          # only clone the policies repo once
+          gcloud source repos clone gcp-policies "${policy_file_path}" --project="${project_id}" || exit 34
+          touch /workspace/.policies-cloned
```

Comment:
Are we using this dir as a marker just for ensuring that the policies have been cloned?
Can't we just clone into a predefined directory `gcloud source repos clone "${policy_file_path}" "${policy_dir}" --project="${project_id}"` and for subsequent runs check for that `policy_dir` exists and is not empty?

Reply:
Yes, /workspace/.policies-cloned is just to check if the repo has been cloned. 

We're already cloning to `$policy_file_path` so I can change line 141 to 
```bash
if [ -n "$(ls -A ${policy_file_path})" ] ; then ...
```
We don't need to check if the directory exists since we already know it does (line 133)

Reply:
yup that makes sense to me